packages/http/src/webauthn-json/base64url.ts

```diff
@@ -17,7 +17,7 @@ export function base64urlToBuffer(
   for (let i = 0; i < str.length; i++) {
     byteView[i] = str.charCodeAt(i);
   }
-  return buffer;
+  return byteView;
```

Comment:
This is unrelated to this specific change, but up until now this function was essentially just returning an empty buffer.